pwnlib/commandline/checksec.py

```diff
@@ -15,20 +15,19 @@
 parser.add_argument(
     'elf',
     nargs='*',
-    type=argparse.FileType('rb'),
     help='Files to check'
 )
 parser.add_argument(
     '--file',
     nargs='*',
     dest='elf2',
     metavar='elf',
-    type=argparse.FileType('rb'),
     help='File to check (for compatibility with checksec.sh)'
 )
 
 def main(args):
-    files  = args.elf or args.elf2 or []
+    files = args.elf or args.elf2 or []
+    files = [open(f, 'rb') for f in filter(os.path.isfile, files)]
```

Comment:
TBH we can still have strings in `files` array, and just pass them on to ELF constructor (it can handle path strings as well as file objects). `FileType('rb')` was merely a means to show prettier errors.

Less code is better code. :wink: